apps/admin/pulumi/index.ts

```diff
@@ -10,11 +9,50 @@ export = async () => {
         WbyEnvironment: String(process.env.WEBINY_ENV)
     });
 
-    const app = new App();
-    const cloudfront = new Cloudfront({ appS3Bucket: app.bucket });
+    const bucket = new CloudFrontBucket("admin-app", {
+        bucketConfig(config) {
+            config.versioning = {
+                enabled: true
+            };
+        }
```

Comment:
This is how you can override any resource configuration. You have some smart defaults, but you can change anything.